compiler/frontend/spark/src/main/java/edu/snu/nemo/compiler/frontend/spark/transform/CollectTransform.java

```diff
@@ -22,44 +22,49 @@
 import java.io.FileOutputStream;
 import java.io.ObjectOutputStream;
 import java.util.ArrayList;
-import java.util.Iterator;
+import java.util.List;
 
 /**
  * Collect transform.
  * @param <T> type of data to collect.
  */
 public final class CollectTransform<T> implements Transform<T, T> {
   private String filename;
+  private FileOutputStream fos;
```

Comment:
Why do we need to have these `fos` and `oos` as variables? It can be created only in `close()`.